pyproject.toml

```
@@ -100,3 +100,12 @@ dev = [
 github-actions = ["pytest-github-actions-annotate-failures"]
 graph = ["networkx"]
 mqf2 = ["cpflows"]
+
+[tool.setuptools]
+py-modules = ["pytorch_forecasting"]
+
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+  "setuptools>61",
```

I think a new vulnerability was recently discovered in `setuptools` (ref. CVE-2024-6345), a newer version may be better.

```suggestion
  "setuptools>=70.0.0",
```